frontend/src/entities/dataset/dataset.tsx

```diff
@@ -65,14 +87,14 @@ export const Dataset = () => {
                     actions={DatasetActions}
                     focusProps={{ createDatasetRef: createDatasetRef }}
                     allItems={datasetList}
-                    columnDefinitions={defaultColumns}
-                    visibleColumns={visibleColumns}
+                    columnDefinitions={isAdmin ? adminDatasetColumns : defaultColumns }
+                    visibleColumns={isAdmin ? visibleAdminColumns : visibleColumns}
                     visibleContentPreference={visibleContentPreference}
                     trackBy='location'
                     itemNameProperty='name'
                     loadingItems={loadingDatasets}
                     loadingAction={loadingAction}
-                    serverFetch={getDatasetsList}
+                    //serverFetch={isAdmin ? getAllDatasets : getDatasetsList}
```

Comment:
Should this be commented out or just not exist?

Reply:
I meant to talk about this in dev sync, I'm trying to get serverFetch to work with `getAllDatasets` or `getDatasetsList` but it's erroring because of typing. Wanted to see if you or Evan knew a way to get this prop to be happy with either function